src/api/migrations/20200622165311-create-commentReaction-table.js

```diff
@@ -0,0 +1,38 @@
+export default {
+    up: (queryInterface, Sequelize) => queryInterface.createTable('CommentReaction', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      commentId: {
+        allowNull: false,
+        type: Sequelize.INTEGER,
+        references: { model: 'Comments', key: 'id' }
+      },
+      userId: {
+        allowNull: false,
+        type: Sequelize.INTEGER,
+        references: { model: 'Users', key: 'id' }
+      },
+      likes: {
+        type: Sequelize.INTEGER
+      },
+      dislikes: {
+        type: Sequelize.INTEGER
+      },
+      reported: {
```

Comment:
@MCFrank16 What does reported means?

Reply:
this will equal to 1 if the user clicks on report comment button